src/shared/components/getting-started-section/index.js

```diff
@@ -21,46 +20,40 @@ import {
 import hexagonsSvg from 'shared/media/images/hexagons.sprite.svg'
 import styles from './index.module.css'
 
-const CID_LENGTH = 46
```

Comment:
Removed in favour of `is-ipfs` because string length is a terrible check for whether something is a CID 😂